templates/settings.yaml.erb

```diff
@@ -114,3 +114,10 @@
     :namespace: foreman
 <%   end -%>
 <% end -%>
+
```

Comment:
This will result in an empty line at the end and restarts for everyone. Can we avoid this?

Reply:
by teaching Puppet to diff YAML by content, and not by checksum? sure ;-)

other than that: there is surely a weird `<%--+-!` invocation to force this, but I can never remember how exactly in ERB.
I can also just move the next line one up

Reply:
also, will it really? the line above is `<% end -%>` and I would think the `-%>` should eat up any empty newlines?